tests/modules/nf-core/annotsv/annotsv/main.nf

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ANNOTSV_ANNOTSV } from '../../../../../modules/nf-core/annotsv/annotsv/main.nf'
+
+workflow test_annotsv_annotsv {
+    
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file(params.test_data["homo_sapiens"]["illumina"]["test_sv_vcf"], checkIfExists: true),
+        file(params.test_data["homo_sapiens"]["illumina"]["test_sv_vcf_tbi"], checkIfExists: true)
+    ]
+
+    annotations = [
+        [ id: 'annotations' ],
+        [] // For stub use only, this will fail if the actual module is run like this
```

Comment:
Is this actually testing the module at all, or just a stub?

Reply:
It's a stub. The annotation files are 20 GB so there's no way I can add them to the test datasets repo, but I tested locally and it works like a charm